pkgs/applications/misc/inochi2d/generic.nix

```
@@ -99,6 +100,7 @@ buildDubPackage (
       . gentl.sh
 
       # Use the fake git to generate version info
+      dub build --config=update-version
       dub build --skip-registry=all --compiler=ldc2 --build=release --config=meta
```

```suggestion
      dub build --skip-registry=all --compiler=ldc2 --build=release --config=update-version
```
The meta task is no longer necessary, it's windows-only logic.

---

Done